pkg/registry/core/shoot/strategy.go

```diff
@@ -87,6 +87,8 @@ func (shootStrategy) PrepareForUpdate(_ context.Context, obj, old runtime.Object
 
 	// TODO(acumino): Drop this after v1.83 has been released.
 	removeDuplicateExtensions(newShoot)
+	// TODO(dimitar-kostadinov): Drop this after v1.83 has been released.
+	removeDuplicateServiceAccountIssuers(newShoot)
```

Comment:
Duplicate service account publishers cause `kube-apiserver` to fail to start. In this case, it's better to avoid `Shoot` creation instead of silently fixing the faulty configuration.

Reply:
Pls see discussion in https://github.com/gardener/gardener/pull/8457#pullrequestreview-1621190904 and following. It's the same here. I guess it makes sense to follow the same approach.